src/azul/azulclient.py

```diff
@@ -600,6 +600,14 @@ def message(source: SourceRef):
         messages = map(message, sources)
         self.queue_mirror_messages(messages)
 
+    def is_valid_failed_queues_state(self) -> bool:
```

Comment:
```suggestion
    def is_valid_fail_queues_state(self) -> bool:
```

We use the name `fail queue` everywhere else.